website/content/en/docs/golang/monitoring/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Golang Based Operator Monitoring
+linkTitle: Monitoring
+weight: 100
+---
```

Comment:
^ Rename menu metrics to Monitoring to keep the same doc nomenclature and following the ing standard for titles.